histree-frontend/package.json

```diff
@@ -38,5 +39,8 @@
     "tslib": "^2.4.0",
     "typescript": "^4.8.4",
     "vite": "^3.1.0"
+  },
+  "dependencies": {
```

Comment:
Do you mind checking if this can be installed as a dev dependency? So basically `npm i -D <package name>`